tests/common/request.py

```diff
@@ -42,7 +42,7 @@ def get_transaction_by_id(transaction_id: str):
 
 
 def post_request_and_wait_for_finalization(
-    payload: dict, interval: int = 5, retries: int = 15
+    payload: dict, interval: int = 10, retries: int = 15
```

Comment:
extra: I thought that it was due to timeout at first. 10 seconds seems like a sensible default so I didn't rolled it back